xcixor/my-ride
# Tests

#### Purpose of this PR
The purpose of this PR is to write failing tests that define the desired features of the application

#### Tasks to be completed
* Write tests for api endpoints 
* Write tests for Ride model
* Write tests for User mode
* Write tests for Controller 

#### Manual Testing
* Clone this repo
* Check out to the tests branch 
* Create a virtual env and install (pytests)[https://docs.pytest.org/en/3.2.5/getting-started.html]
* Run pytest tests/ to run all the tests in the tests directory

#### Relevant Pivotal Tracker stories
- [Write tests for application #158575425](https://www.pivotaltracker.com/n/projects/2179559)
